default/config-overrides.js

```diff
@@ -1,9 +1,16 @@
 const rewireYAML = require('react-app-rewire-yaml');
-const { override, removeModuleScopePlugin } = require('customize-cra');
+const { override, removeModuleScopePlugin, addExternalBabelPlugins } = require('customize-cra');
 
 function overrides(config, env) {
 	config = rewireYAML(config, env);
 	return config;
 };
 
-module.exports = override(overrides, removeModuleScopePlugin());
\ No newline at end of file
+module.exports = override(
+	overrides,
+	removeModuleScopePlugin(),
+	...addExternalBabelPlugins(
+		"@babel/plugin-proposal-optional-chaining",
+		"@babel/plugin-proposal-nullish-coalescing-operator"
```

Comment:
why do we need this? What this plugin is doing exactly?

Reply:
In `algob` we are using `??` operator. In `tsconfig` we are targeting EcmaScript version 2020, which webpack doesn't support yet. So importing algob will throw error: as webpack won't be able to parse it. Issue - [https://github.com/facebook/create-react-app/issues/9468](https://github.com/facebook/create-react-app/issues/9468). So we needed to add these plugins.